lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../player.rb'
+
+class Player::Human < Player
+  def move(board)
+    puts "please make a move from 1-9"
+    input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "please make a move from 1-9"
  gets.chomp
end
```